locales/ku/LC_MESSAGES/duckduckgo.po

```diff
@@ -6106,34 +6137,34 @@ msgid "Who We Are"
 msgstr "Em Kî Ne"
 
 msgid "Why Privacy"
-msgstr ""
+msgstr "Çima Taybetî"
 
 msgctxt "image-layout"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. http://i.imgur.com/twR2utt.png
 msgctxt "settingsvalue"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. http://i.imgur.com/SOkVdkM.png
 msgctxt "width"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. https://duckduckgo.com/params under "Look & Feel Settings"
 msgid "Width:"
-msgstr ""
+msgstr "Ferehî"
 
 msgctxt "feedback form"
 msgid "Wikipedia info"
-msgstr ""
+msgstr "Zanyariyên Wikipedia"
 
 #. Element title explaining this is the win loss record for team, e.g. 10-5
 msgctxt "Sports module"
 msgid "Win/Loss Record"
-msgstr ""
+msgstr "Encama "
```

Comment:
This looks incomplete. Do you mind double checking?

Reply:
done